ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ghostFS:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      PerformanceMode: generalPurpose
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
+  ghostAPI:
+    Type: AWS::ECS::TaskDefinition
+    Properties:
+      ContainerDefinitions:
+        - Image: registry.hub.docker.com/library/ghost
+          PortMappings:
+            - ContainerPort: 2368
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref ghostAPILogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: logs
+          MountPoints:
+            - 
+              ContainerPath: /var/lib/ghost/content
+              SourceVolume: ghostVolume
+          Name: 0
+          Environment:
+            - Name: url
+              Value: !Join [ '', [ 'http://', !GetAtt Loadbalancer.DNSName ] ]
+            # - Name: database__client
+            #   Value: sqlite3
+            # - Name: database__connection__filename
+            #   Value: '/var/lib/ghost/content/data/ghost-efs.db'
+            - Name: paths__contentPath
+              Value: /var/lib/ghost/content
+            - Name: DB_ARN
+              Value: !Sub arn:aws:rds:${AWS::Region}:${AWS::AccountId}:cluster:${ghostDB}
+            - Name: DB_ROOT_USER_SECRET_ARN
+              Value: !Ref ghostDBRootUserSecret
```

Comment:
I suppose you could delete the above two env vars as they don't appear to be used.

Reply:
I believe I included them for debugging purposes early on, bc it's easy to view things through the task definition. But I think you're right that they're dormant.